Gemfile

```diff
@@ -76,6 +76,7 @@ group :test do
   gem 'rspec-json_expectations'
   gem 'rspec_junit_formatter'
   gem 'rspec-rails'
+  gem 'sequel_spec'
```

Comment:
Mildly cautious of including a gem which hasn't been changed in 5 years and its last significant change was 6 years ago.

It does allow writing much nicer validation specs though so :man_shrugging: 

An alternative for the validations - not as nice or thorough but still fairly compact with discrete tests for different attributes
* https://github.com/trade-tariff/trade-tariff-backend/blob/main/spec/models/news/item_spec.rb#L18